packages/zosjobs/__tests__/__system__/CancelJobs.system.test.ts

```diff
@@ -43,80 +43,80 @@ describe("CancelJobs System tests", () => {
         const ACCOUNT = systemProps.tso.account;
         const maxStepNum = 255;  // Use lots of steps to make the job stay in INPUT status longer
```

Comment:
I know we need the job to stay running for at least 10-20 seconds to give us adequate time to cancel it. 👍 

Should we consider making `maxStepNum` smaller? Currently if a CancelJobs test fails, it could leave the job running for 255*10 seconds ~= 42 minutes, which seems longer than necessary.

Reply:
I am going to change this to run for, at most, 60 seconds, just in case.